CHANGES.md

```diff
@@ -1,3 +1,28 @@
+# snare 0.4.0 (2020-xx-xx)
+
+## Breaking changes
+
+* The `email` option in `match` blocks has been replaced by the more generic
+  `errorcmd`. To obtain the previous behaviour:
+
+    ```
+    email = "someone@example.com";
+    ```
+
+  should be changed to:
```

Comment:
should -> could?

There's any number of mail clients you could use, or I suppose you could even invoke sendmail directly.

Reply:
I think "could" is just unnecessarily vague. If someone realises there's a better way for them, they'll find it. Everyone else can copy and paste this almost unthinkingly and be all-but-sure it'll work.

Reply:
> Everyone else can copy and paste this almost unthinkingly and be all-but-sure it'll work.

I disagree. `mailx` is not standard. For example, I have a fresh Debian install here, and it has neither `mailx` nor `mail`.

Reply:
Every box I've seen that has an active sendmail has `mailx`. So if the `email` options worked before, I'm relatively confident that `mailx` inside `errorcmd` will work now.

Reply:
OK, so I just installed sendmail on the Debian box. Still no `mailx` or `mail` :P

Really, I don't think we should encourage copy-pasting.

I'm not going to die on this hill, so if you still disagree, I'll back down.

Reply:
OK, how about I expand it to "if you have `mailx` installed"?

Reply:
Sure.

Reply:
Fixed in https://github.com/softdevteam/snare/pull/65/commits/54c6ad864a2809931c57000c8e5c3ac7c2b53910.